templates/statefulset.yaml

```diff
@@ -1,7 +1,351 @@
+{{- if .Values.ApiService.enabled }}
```

Comment:
This separation looks weird. We don't need two StatefulSets to achieve the goal.
If it works - let's merge it for now and I'll refactor it later.